src/components/ha-form/compute-initial-ha-form-data.ts

```diff
@@ -0,0 +1,37 @@
+import { HaFormSchema } from "./types";
+
+export const computeInitialHaFormData = (
+  schema: HaFormSchema[]
+): Record<string, any> => {
+  const data = {};
+  schema.forEach((field) => {
+    if (field.description?.suggested_value) {
+      data[field.name] = field.description.suggested_value;
```

Comment:
Doesn't this mean that we can never not set an optional field with `suggested_value`?